packages/web3-types/src/eth_types.ts

```diff
@@ -409,7 +409,8 @@ export interface PopulatedUnsignedBaseTransaction {
 	gas?: Numbers;
 	gasPrice: Numbers;
 	type: Numbers;
-	input: Bytes;
+	input?: Bytes;
+	data?: Bytes;
```

Comment:
This is a breaking change for `eth_types`, but `input` won't be apart of this object if the user uses `data`